config/locales/api/sid_ET.yml

```diff
@@ -111,11 +111,11 @@ sid-ET:
         chapter_7_html: "7: Hassiisseeme sonubba"
         chapter_8_html: "8: Duucha woyte Xa'mantino xa'mubba"
         chapter_9_html: "9: Hiittoonni kaa’lo afira dandiinayiro"
-        chapter_10_html: "10: Simple horonsirateyi hanafi"
+        chapter_10_html: "10: Shota horonsirateyi hanafi"
```

Comment:
Do we want to localize the word `Simple`?

Reply:
Let's ignore these changes. You can ask @harimohanraj89 to help flag this to GPI (our translations agency) so they keep this in mind in the future.